src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { mount } from "enzyme"
+import React from "react"
+import { commitMutation, RelayProp } from "react-relay"
+
+import { Button } from "Styleguide/Elements/Button"
+import { Radio } from "Styleguide/Elements/Radio"
+import { Provider } from "../../../../../node_modules/unstated"
```

Comment:
I suspect this was from VSCode auto-import; should be 
```
import { Provider } from "unstated"
```

(I wonder if this is a bug in VSCode or a setting that needs to be adjusted -- this has happened quite a bit to me too)

Reply:
Ha, yeah it's definitely an auto-import issue. Not sure if it's a bug or config issue.

Reply:
Hrm, it looks like this _should_ be fixed with TS 3.0 - but maybe it isn't? 
https://github.com/Microsoft/TypeScript/issues/23230#issuecomment-404330449